config/resources/government.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "kans": "http://kanselarij.vo.data.gift/core/",
+    "skos": "http://www.w3.org/2004/02/skos/core#",
+    "ext": "http://mu.semte.ch/vocabularies/ext/"
+  },
+  "resources": {
+    "government-fields": {
+      "new-resource-base": "http://themis.vlaanderen.be/beleidsveld/",
```

Comment:
Om te voldoen aan de URI-strategie van Vlaanderen moet de URI een structuur hebben als `<domain-name>/id/<model-name-enkelvoud-in-NL>/`, dus in dit geval `http://themis.vlaanderen.be/id/beleidsveld/`. Deze opmerking komt nog op een aantal plaatsen voor in deze file.